Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM python:2.7.12
```

Comment:
It can wait till there's time. But xenial has 2.7.11 and this code was written for 2.6, so it shouldn't need any changes.